examples/apis/order/updatetransaction/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/order"
+)
+
+func main() {
+	accessToken := "{{Access_token}}"
+	c, err := config.New(accessToken)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	client := order.NewClient(c)
+	orderID := "{{EXISTING_ORDER_ID}}"
+	transactionID := "{{EXISTING_TRANSACTION_ID}}"
+
+	// // Updating the transaction to change the number of installments
```

Comment:
[req] `//` de comentário duplicada
```suggestion
	// Updating the transaction to change the number of installments
```